tests/CheckDen.py

```diff
@@ -12,7 +12,7 @@
 import sys
 sys.path.append('../')
 
-from lookups import Util
+from lookups import Util,PKMString
```

Comment:
Why adding this?

Reply:
for nature filtering, isn't it necessary? PKMString().natures[r.Nature] == 'Careful' for example

Reply:
`Util `has a global variable `Util.STRINGS.natures` and it can be translated by `Util.translate('it')`